back/app/services/permissions/idea_permissions_service.rb

```diff
@@ -43,43 +44,37 @@ def action_descriptors(idea, user)
           up: {
             enabled: !liking_disabled_reason,
             disabled_reason: liking_disabled_reason,
-            future_enabled_at: liking_disabled_reason && future_enabled_phase('reacting_idea', user, idea.project, reaction_mode: 'up')&.start_at
+            future_enabled_at: liking_disabled_reason && project_permissions_service.future_enabled_phase('reacting_idea', reaction_mode: 'up')&.start_at
           },
           down: {
             enabled: !disliking_disabled_reason,
             disabled_reason: disliking_disabled_reason,
-            future_enabled_at: disliking_disabled_reason && future_enabled_phase('reacting_idea', user, idea.project, reaction_mode: 'down')&.start_at
+            future_enabled_at: disliking_disabled_reason && project_permissions_service.future_enabled_phase('reacting_idea', reaction_mode: 'down')&.start_at
           }
         },
         comment_reacting_idea: {
           enabled: !comment_reacting_disabled_reason,
           disabled_reason: comment_reacting_disabled_reason,
-          future_enabled_at: comment_reacting_disabled_reason && future_enabled_phase('commenting_idea', user, idea.project)&.start_at
+          future_enabled_at: comment_reacting_disabled_reason && project_permissions_service.future_enabled_phase('commenting_idea')&.start_at
         },
         voting: {
           enabled: !voting_disabled_reason,
           disabled_reason: voting_disabled_reason,
-          future_enabled_at: voting_disabled_reason && future_enabled_phase('voting', user, idea.project)&.start_at
+          future_enabled_at: voting_disabled_reason && project_permissions_service.future_enabled_phase('voting')&.start_at
         }
       }
     end
 
     private
 
-    def idea_in_current_phase?(idea, current_phase)
-      idea.ideas_phases.find { |ip| ip.phase_id == current_phase.id }
-    end
+    attr_reader :idea
 
-    def posting_limit_reached?(phase, user)
```

Comment:
The same method exists in the phase permissions service, but there seems to be a subtle difference. Which one is the more correct one?